files/rsyslog/rsyslog.sh

```diff
@@ -107,8 +107,10 @@ if [ -z $ES_PORT ]; then
 fi
 
 # Check bearer_token_file for fluent-plugin-kubernetes_metadata_filter.
-if [ ! -s /var/run/secrets/kubernetes.io/serviceaccount/token ] ; then
-    echo "ERROR: Bearer_token_file (/var/run/secrets/kubernetes.io/serviceaccount/token) to access the Kubernetes API server is missing or empty."
+k8s_token="/var/run/secrets/kubernetes.io/serviceaccount/token"
+k8s_cacert="/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+if [ ! -s "${k8s_token}" -o ! -s "${k8s_cacert}" ] ; then
+    echo "ERROR: Bearer_token_file (${k8s_token}) or ca cert (${k8s_cacert}) to access the Kubernetes API server is missing or empty."
```

Comment:
one consequence of this is that QE is going to report that rsyslog doesn't start at all and gives this error message - we'll need to catch that and close them as duplicates of 1745394 - I suppose rsyslog not starting at all (fail fast) is better than rsyslog starting but not actually working (fail late).